packages/nextjs/src/client/index.tsx

```diff
@@ -39,7 +51,9 @@ export function ClerkProvider({ children, ...rest }: NextClerkProviderProps): JS
       publishableKey={publishableKey || process.env.NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY || ''}
       clerkJSUrl={clerkJSUrl || process.env.NEXT_PUBLIC_CLERK_JS}
       // @ts-expect-error
-      proxyUrl={proxyUrl || process.env.NEXT_PUBLIC_CLERK_PROXY_URL}
+      proxyUrl={proxyUrl || process.env.NEXT_PUBLIC_CLERK_PROXY_URL || ''}
+      domain={domain || ''}
```

Comment:
🔧 We should introduce NEXT_PUBLIC_CLERK_DOMAIN and NEXT_PUBLIC_CLERK_IS_SATELLITE.

Reply:
I see your point about a `*CLERK_DOMAIN` env var, but `CLERK_IS_SATELLITE` is supposed to be a boolean. I am not a big fan of converting strings to booleans. I think the `isSatellite` is sufficient.